### ISSUE TITLE
Add support for double colon rules

### ISSUE THREAD

**Issue by mbaldi** [carlos-algms/vscode-make-task-provider#29]
Hey there, thanks for the hard work
Would you consider adding support for [double colon rules ](https://www.gnu.org/software/make/manual/html_node/Double_002dColon.html)?
like 

> `default :: build`

They seem not being parsed at the moment

**Comment by carlos-algms** [comment]
Hello, thanks for requesting this feature. 

It will be implemented by #30 

Do you have any specific or tricky examples I should cover as well?

**Comment by mbaldi** [comment]
This is great, thanks!

The examples that I have are pretty basic

```
default :: build

build ::
        ./run-build.sh

clean ::
        rm -rf build

all :: clean default
```
